C/CUDA/CUDA_full@10.2/build_tarballs.jl

```diff
@@ -133,10 +133,25 @@ elif [[ ${target} == aarch64-linux-gnu ]]; then
     find $temp
     mv $temp/usr/local/cuda-10.2/* ${prefix}/cuda
 
-    mv -nv $temp/usr/include/* ${prefix}/cuda/targets/aarch64-linux/include
-    mv -nv $temp/usr/lib/$target/* ${prefix}/cuda/targets/aarch64-linux/lib
+    rsync -aOv --remove-source-files $temp/usr/include/ ${prefix}/cuda/targets/aarch64-linux/include/
```

Comment:
Maybe add a symlink then?

Reply:
The Debian packages install stuff in `/usr/{include,lib}` (e.g. `cublas`) - in addition to `/usr/local/cuda-10.2` - it's a mess. So I'm moving the stuff from both `/usr/local/cuda-10.2` and `/usr/{include,lib}` into `$prefix/cuda`

Reply:
... I only replaced the `mv`'ing about with `rsync` as there were some stub `so`'s in `$temp/usr/lib/$target/stubs` that was not being moved into place due to an existing `${prefix}/cuda/targets/aarch64-linux/lib/stubs` directory.